frontend/src/pages/Registration.tsx

```diff
@@ -127,13 +129,25 @@ export const Registration: React.FC = (): JSX.Element => {
             <FormControl isRequired marginTop={6}>
```

Comment:
Need to show invalid password message from backend on Submit attempt
```suggestion
            <FormControl isRequired marginTop={6} isInvalid={formatUserDetailsKey.password in errorMap}>
```